src/network/networkd-manager.c

```diff
@@ -622,11 +624,13 @@ int manager_rtnl_process_address(sd_netlink *rtnl, sd_netlink_message *message,
         case RTM_DELADDR:
 
                 if (address) {
-                        log_link_debug(link, "Removing address: %s/%u (valid %s%s)", buf, prefixlen,
+                        log_link_debug(link, "Forgetting address: %s/%u (valid %s%s)",
+                                       strnull(buf), prefixlen,
                                        valid_str ? "for " : "forever", strempty(valid_str));
                         (void) address_drop(address);
                 } else
-                        log_link_warning(link, "Removing non-existent address: %s/%u (valid %s%s), ignoring", buf, prefixlen,
+                        log_link_warning(link, "Kernel removed an address we don't remember: %s/%u (valid %s%s), ignoring.",
```

Comment:
I think the log level here can be lower. Maybe `LOG_DEBUG` or `LOG_INFO` is enough.

Reply:
I'll use `log_info`. If it turns out to be annoying, we can lower later.